src/components/admin/UserProfileModal/components/ProfileModalQuestions.tsx

```diff
@@ -32,24 +33,28 @@ export const ProfileModalQuestions: React.FC<ProfileModalQuestionsProps> = ({
           className="ProfileModalQuestions__question-container"
           key={question.text}
         >
-          <div className="ProfileModalQuestions__question">{question.text}</div>
-          {editMode && register ? (
-            <ProfileModalInput
-              containerClassName="ProfileModalQuestions__answer-input"
-              defaultValue={answers[i]}
-              register={register}
-              name={question.name}
-            />
-          ) : (
-            <div className="ProfileModalQuestions__answer">{answers[i]}</div>
-          )}
+          <InputGroup title={question.text}>
+            {editMode && register ? (
+              <Input
+                defaultValue={answers[i]}
+                register={register}
+                name={question.name}
+              />
+            ) : (
+              <div className="ProfileModalQuestions__answer">
```

Comment:
Can we remove `ProfileModalQuestions__answer`?

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8